openedx/core/djangoapps/credit/migrations/0008_auto__del_index_creditprovider_provider_id__del_field_crediteligibilit.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Deleting field 'CreditEligibility.provider'
+        db.delete_column('credit_crediteligibility', 'provider_id')
+
+
+    def backwards(self, orm):
+        # User chose to not deal with backwards NULL issues for 'CreditEligibility.provider'
+        raise RuntimeError("Cannot reverse this migration. 'CreditEligibility.provider' and its values cannot be restored.")
```

Comment:
@aamir-khan We need to be able to run migrations backwards in case of a rollback.

I'm not sure I understand why this is necessary.  When I checked out your branch locally I was able to run forwards and backwards migrations successfully (with the `RuntimeError` removed):
```
edxapp@precise64:~/edx-platform$ ./manage.py lms migrate credit 0008 --settings=devstack
 - Soft matched migration 0008 to 0008_auto__del_index_creditprovider_provider_id__del_field_crediteligibilit.
Running migrations for credit:
 - Migrating forwards to 0008_auto__del_index_creditprovider_provider_id__del_field_crediteligibilit.
 > credit:0008_auto__del_index_creditprovider_provider_id__del_field_crediteligibilit
 - Loading initial data for credit.
Installed 0 object(s) from 0 fixture(s)
edxapp@precise64:~/edx-platform$ ./manage.py lms migrate credit 0007 --settings=devstack
 - Soft matched migration 0007 to 0007_auto__add_field_creditprovider_enable_integration__chg_field_creditpro.
Running migrations for credit:
 - Migrating backwards to just after 0007_auto__add_field_creditprovider_enable_integration__chg_field_creditpro.
 < credit:0008_auto__del_index_creditprovider_provider_id__del_field_crediteligibilit
edxapp@precise64:~/edx-platform$ ./manage.py lms migrate credit 0008 --settings=devstack
 - Soft matched migration 0008 to 0008_auto__del_index_creditprovider_provider_id__del_field_crediteligibilit.
Running migrations for credit:
 - Migrating forwards to 0008_auto__del_index_creditprovider_provider_id__del_field_crediteligibilit.
 > credit:0008_auto__del_index_creditprovider_provider_id__del_field_crediteligibilit
 - Loading initial data for credit.
Installed 0 object(s) from 0 fixture(s)
```

Are you sure that the error you're seeing is a problem with the migration, not the state of your local database?